python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -222,6 +275,10 @@ async def config_callback(self, client):
             followingErrorThreshold=client.config.following_error_threshold,
             trackingSuccessPositionThreshold=client.config.track_success_pos_threshold,
             trackingLostTimeout=client.config.tracking_lost_timeout,
+            maxVelocityLimit=client.config.max_velocity_limit,
+            emergencyAccelerationLimit=client.config.emergency_accel_limit,
+            emergencyJerkLimit=client.config.emergency_jerk_limit,
+            disableLimitMaxTime=client.config.disable_limit_max_time,
```

Comment:
Different order compared with the update at `class Config(ctypes.Structure)`.

Reply:
Yes. I am using the order of fields in the MTRotator config event (which is different than the order in the config data for the low-level controller because I tried to make a more logical order).